src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using `@BeforeEach` instead of `@BeforeAll` for initializing `RegistrationService`. `@BeforeAll` is used for static initializations that are executed once before all tests, while `@BeforeEach` runs before each test, ensuring test isolation.